src/components/SignatureRequest.js

```diff
@@ -0,0 +1,186 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { Button, GU, textStyle, useTheme } from '@aragon/ui'
+import { useWallet } from '../providers/Wallet'
+import { signMessage } from '../lib/web3-utils'
```

Comment:
ah, the lib one i think that is more mapped with real libs like web-, ipfs but we can remove that folder if you want

Reply:
Gotcha, it did seem like the utils was more directly related to the court? But yes, might be good to just use one for now, since we have other utils like `date-utils` in `/utils`